soccer/planning/DirectPathPlanner.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include "planning/Path.hpp"
+#include "planning/PathPlanner.hpp"
+
+namespace Planning {
+    class DirectPathPlanner : public PathPlanner {
```

Comment:
Well its going to give direct paths. Eventually. Once we finish refactoring the pathplanners to be sort of interchangable.